README.md

```diff
@@ -194,6 +194,176 @@ To use the Flow Token contract as is, you need to follow these steps:
+              ?? panic("Could not borrow reference to switchboard")
+        }
+
+        execute {
+          // Add the capability to the switchboard using addNewVault method
+          self.switchboardRef.addNewVaultsByPath(paths: self.vaultPaths, 
+                                                        address: address)
+        }
+    }
+    ```
+    This function won't panic, instead it will just not add to the `@Switchboard` any capability which can not be retrieved from any of the provided `PublicPath`s. It will also ignore any type of `&{FungibleToken.Receiver}` that is already present on the `@Switchboard`
+
+ ## Removing a vault from the switchboard
+ If a user no longer wants to be able to receive deposits from a certain FT, or if they want to update the provided capability for one of them, they will need to remove the vault from the switchboard. This can me acomplish by using `removeVault(capability: Capability<&{FungibleToken.Receiver}>)`. As for adding new vaults it is required to first retrieve the vault's capability from the user account and passing it as a parameter. This can be observed in the template transaction `transactions/switchboard/remove_vault_capability.cdc`:
+ ```cadence
+ transaction {
+    let exampleTokenVaultCapabilty: Capability<&{FungibleToken.Receiver}>
+    let switchboardRef:  &FungibleTokenSwitchboard.Switchboard
+
+    prepare(signer: AuthAccount) {
+      // Get the example token vault capability from the signer's account
+      self.exampleTokenVaultCapabilty = signer.getCapability
+                    <&{FungibleToken.Receiver}>(ExampleToken.ReceiverPublicPath)
+      // Get a reference to the signers switchboard  
+      self.switchboardRef = signer.borrow<&FungibleTokenSwitchboard.Switchboard>
+        (from: FungibleTokenSwitchboard.StoragePath) 
+          ?? panic("Could not borrow reference to switchboard")
+
+    }
+
+    execute {
+      // Remove the capability from the switchboard using the 
+      // removeVault method
+      self.switchboardRef.removeVault(capability: self.exampleTokenVaultCapabilty)
+    }
+ }
+ ```
+ This function will panic if is not possible to `.borrow()` a reference to a `&{FungibleToken.Receiver}` from the passed capability.
+
+ ## Transfering tokens through the switchboard
+ The Fungible Token Switchboad provides two different ways of depositing tokens to it, using the `deposit(from: @FungibleToken.Vault)` method enforced by the `{FungibleToken.Receiver}` or using the `safeDeposit(from: @FungibleToken.Vault): @FungibleToken`:
+
+ 1. Using the first method will be just the same as depositing to `&{FungibleToken.Receiver}`. The path for the Switchboard receiver is defined in `FungibleTokenSwitchboard.ReceiverPublicPath`, the generic receiver path `/public/GenericFTReceiver` that can also be obtained from the NFT MetadataViews contract. An example of how to do this could be find in the transaction template on this repo `transactions/switchboard/transfer_tokens.cdc`
```

Comment:
```suggestion
 1. Using the first method will be just the same as depositing to `&{FungibleToken.Receiver}`. The path for the Switchboard receiver is defined in `FungibleTokenSwitchboard.ReceiverPublicPath`,
 the generic receiver path `/public/GenericFTReceiver` that can also be obtained from the NFT MetadataViews contract.
 An example of how to do this can be found in the transaction template on this repo `transactions/switchboard/transfer_tokens.cdc`
```